jenkins/mergerequest

```diff
@@ -1,16 +1,15 @@
 pipeline {
     agent {
         node {
-            label 'daint'
+            label 'balfrin'
         }
     }
 
     stages {
         stage('Pytest') {
             steps {
                 sh'''
-                source /project/g110/icon/probtest/conda/miniconda/bin/activate
-                conda activate probtest
+                source $SCRATCH/../jenkins/project/g110/icon/probtest/conda/miniconda/bin/activate probtest
```

Comment:
Isn't there a direct path to the env?
Why first go back with `../` to only go into `jenkins` again?

Reply:
I like to use the $SCRATCH variable her, as the absolute path has changed in the past on Balfrin. The `../` is such that it works for every user. But as it here is only meant to be the jenkins user, I am fine to remove it.